scripts/parser-tests/typescript/index.js

```diff
@@ -1,37 +1,58 @@
 import path from "path";
-import fs from "fs/promises";
+import fs from "fs";
 import { fileURLToPath } from "url";
-import ts from "../../../build/typescript/lib/typescript.js";
 import TestRunner from "../utils/parser-test-runner.js";
-import parsingErrorCodes from "./error-codes.js";
+import ErrorCodes from "./error-codes.js";
+
+import { spawnSync } from "child_process";
+const getEncoding = path =>
+  spawnSync("file", ["--brief", "--mime-encoding", path])
+    .stdout.toString()
+    .slice(0, -1);
```

Comment:
If this is for a trailing newline, `.trim()` is more explicit.

Reply:
Done: https://github.com/babel/babel/pull/14320/commits/8e16a07f4b2698b85a5e2ee9f3edf2c22cb7631a